tests/test_batch_reconstruction.py

```diff
@@ -96,6 +97,70 @@ def handle_async_exception(loop, ctx):
         for elem in r:
             assert type(elem) is GFElement
 
+
+@mark.asyncio
+async def test_fft():
+    n = 4
+    p = 97
```

Comment:
Can you change it to `elliptic_curve.BLS...`? It is a larger modulus and we have it in most places in the existing code.

Reply:
Sure